tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1231,13 +1231,18 @@ a dictionary can also be used with a named enum.
 
     # Is the same as:
     const State = {STATE_IDLE = 0, STATE_JUMP = 5, STATE_SHOOT = 6}
+    # Access values with State.STATE_IDLE, etc.
 
     func _ready():
         # Access values with Name.KEY, prints '5'
         print(State.STATE_JUMP)
         # Use constant dictionary functions
         # prints '["STATE_IDLE", "STATE_JUMP", "STATE_SHOOT"]'
         print(State.keys())
+        # prints '["STATE_IDLE", "STATE_JUMP", "STATE_SHOOT"]'
+        print(State)
```

Comment:
Wait...

This is not true. `State` is a Dictionary so it should print `{ "STATE_IDLE": 0, "STATE_JUMP": 5, "STATE_SHOOT": 6 }`.